ibm_catalog.json

```diff
@@ -112,6 +112,7 @@
             {
               "key": "existing_resource_group_name",
               "required": true,
+              "default_value": "Default",
```

Comment:
default values should not be added in the json - they should be inherited from terraform code. Otherwise you have 2 places to maintain